eng/pipelines/runtime.yml

```diff
@@ -117,6 +117,21 @@ extends:
                 eq(stageDependencies.EvaluatePaths.evaluate_paths.outputs['SetPathVars_non_mono_and_wasm.containsChange'], true),
                 eq(variables['isRollingBuild'], true))
 
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+          platforms:
+          - browser_wasm
+          jobParameters:
+            nameSuffix: AllSubsets_CoreCLR
+            buildArgs: -s mono.emsdk+clr.paltests -rc Release -c Release -lc $(_BuildConfig)
```

Comment:
This hardcodes runtime to Release and leaves libraries to "_BuildConfig" - that doesn't seem right, I would expect it the other way round

Reply:
It does it the same way as the coreclr lanes around. Not sure what's the reasoning there.